src/niclips/figures/dwi.py

```diff
@@ -0,0 +1,190 @@
+"""Diffusion MRI figure generation module."""
+
+from pathlib import Path
+
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import nibabel as nib
+import numpy as np
+from matplotlib.animation import FuncAnimation
+from matplotlib.gridspec import GridSpec
+
+import niclips.image as noimg
+from niclips.checks import check_4d
+from niclips.defaults import get_default_coord, get_default_vmin_vmax
+from niclips.figures.multi_view import three_view_video
+from niclips.typing import StrPath
+
+mpl.use("Agg")
```

Comment:
We don't actually need this - I was trying to prevent it from opening windows in the background and address a memory issue, but I think I changed it in a recent commit to explicitly close the figure.